### DISCUSSION TITLE
HTTP/2?

### DISCUSSION THREAD

**Discussion by numbofathma** [vercel/vercel#4569] [Help] [answered]
Hello guys, 

From what I've read Vercel serves on HTTP/2 protocol, but on my portfolio:
https://costinmirica.com/
I get my assets served via HTTP/1.1, any ideas?
This is the GitHub repo for my project: https://github.com/numbofathma/costinmirica

Thanks!

**Comment by markozxuu** [comment, marked as answer]
Hi @numbofathma 👋  ! I'm checking your site and if it has the **HTTP/2** protocol : )

<img width="1276" alt="Screen Shot 2020-06-06 at 21 46 28" src="https://user-images.githubusercontent.com/16585386/83959184-a00f8880-a83f-11ea-8877-b71cc5df6b89.png">

**Reply by numbofathma** [reply]
How did it show you HTTP/2? Hmmm I think I found my bug:
https://stackoverflow.com/questions/54513933/requests-served-from-serviceworker-are-downgraded-from-http-2-to-http-1-1

**Reply by markozxuu** [reply]
We can confirm that it is a cache problem that the services worker has 🤔

**Comment by numbofathma** [comment]
Hello @markozxuu ! Thank you for your answer, this is what I've got on my side:
<img width="1680" alt="Screenshot 2020-06-07 16 12 35" src="https://user-images.githubusercontent.com/4989935/83969717-2bd5f300-a8da-11ea-8777-4148e139fde2.png">
